packages/electron-react-mobx-boilerplate/yarn.lock

```diff
@@ -0,0 +1,9230 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please delete the packages' yarn.lock files. There should only be one in the root since this is a monorepo. Make sure you run yarn install to update that lockfile.